builder/src/scss/general.scss

```diff
@@ -44,7 +49,7 @@
     .image {
         position: absolute;
         width: 100%;
-        height: 100%;
+        height: 50%;
         background-position: center !important;
         background-repeat: no-repeat !important;
         background-size: cover !important;
```

Comment:
`!important` should be removed. The reason you had to include them here is that you have the inline rule `background: url({% static "background.jpg" %});` on `communities_list.html`. The `background` property is a shorthand for all background-related properties (`background-image`, `background-position` etc.), thus overriding any background-related rules in your css-file. If you rename the rule in `communities_list.html` to `background-image` (as you always should when strictly declaring a background-image) instead of `background`, the need to override rules in css with `!important` disappears.